### ISSUE TITLE
Ores not smelting in pit kiln

### ISSUE THREAD

**Issue by Kanaminn43** [TerraFirmaCraft/TerraFirmaCraft#179]
I'm having a very strange issue, I'm in a modpack I've made so I can prepare one with tfc in the future, and for some reason the pit kiln doesn't smelt ores. Whenever I do when it finishes firing the ores are still in there while the temperature is brilliant white. I've even built the pack up again from scratch and had it work again, until I relaunched several times to have it not work again.

Here's the part of the latest log where the error happens (I couldn't upload the whole log)
[Latest Log](https://gist.githubusercontent.com/Kanaminn43/760a7c7d53c9a676ae67b78a2748d5ee/raw/5d1eb6034d7d1a408d7c079490a61904b17e6366/latest%2520log)

aaand here's the mod list
[Mod List](https://gist.githubusercontent.com/Kanaminn43/aebdb7090448d15fa396c7ea452eae20/raw/0877e55e233a359b336d82415ad60f8f919d334e/mod%2520list)

To get even more into detail, I was trying to smelt 10 pieces of tetrahedrite nuggets.

**Comment by alcatrazEscapee** [comment]
That part of the log you uploaded has nothing to do with this behavior and tells me nothing. In addition, explain specifically what you are trying to do: are you trying to place ores directly in the pit kiln?

**Comment by Kanaminn43** [comment]
guess it doesn't say anything then when it happens.

I'm putting the ore in the ceramic vessel, sticking it in the pit kiln, filling it with straw and logs, lighting it, letting it sit. Later finding out there's no liquid in the vessel despite being brilliant white.

**Comment by alcatrazEscapee** [comment]
1. Upload the full log. Use pastebin or other similar sites.
2. Is the vessel itself saying it's brilliant white, or are the contents (the ores) brilliant white as well?
3. Is it not working with specific ores / ingredients, or does it never work?

**Comment by Kanaminn43** [comment]
1. [The log was way too large for pastebin/gist so I had to upload it to google drive.](https://drive.google.com/open?id=1SSzlTT2IYl5eIhx1MZFTY1wjAQQheAiY) 
2. It's just the vessel saying that it's brilliant white, the ore inside says nothing.
3. I have tested every ore & ore grade that is smeltable in the pit kiln, and none of them ever smelt.

**Comment by alcatrazEscapee** [comment]
What size (units) of ores are you using?

**Comment by Kanaminn43** [comment]
I was using the nuggets (10 units.) But none of the other sizes worked either.

**Comment by alcatrazEscapee** [comment]
Two things:
 - I've been informed that nuggets are currently broken, i.e. they can't melt.
 - If there are any other items that don't melt (in this case, nuggets count) in the vessel at the same time, nothing will melt.

**Comment by Kanaminn43** [comment]
I don't quite get what you mean with that second one. Are you saying that if there is something that can't melt i.e, nuggets, in the vessel at the same time as something that can, nothing will melt?

If so, I've tested all sizes separate to avoid that, here's a picture of native copper failing to melt, just to get a visual on what kind of test I was doing.
 
![javaw_07-21-10 24 33](https://user-images.githubusercontent.com/29335214/61592498-1c569a00-aba2-11e9-8971-0ffb93b578a6.png)

It's like that for every ore, but it works in "vanilla" tfc. But some mod interaction has to be messing with it (and it's a mystery since I put one or so mods in at a time so I could catch it, but ended up with all of them in with it working, it wasn't until I relaunched and created a new world when it broke again)
